### PR TITLE
Refactored Error 404

### PR DESCRIPTION
**Pull request by connormckellips**
Aligned Error 404 page with JIRA requirements

## For Pull Requesters/Reviewers:
All of the following should be true before making a pull request and should be inspected by reviewers:

### User Story
- [x] Acceptance criteria are adhered to and completely finished
- [x] All subtasks on Jira are completed

### Unit Testing
- [x] Unit tests are finished and passing
- [x] Code coverage is >80%

### Documentation/Style
- [x] JSDocs are written for methods and components
- [x] Unclear code has been documented
- [x] Style standards are adhered to

## For Reviewers:
The following should be additionally checked by reviewers:

- [x] Code runs locally
- [x] Visually inspect tests; they should not simply pass, but rather they should be doing something and testing acceptance criteria
- [x] See if "jank" code could be rewritten

## Description
Insert here the description of what your PR does.

### CODE REVIEW

File: client/PageRoutes.jsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ export default function PageRoutes() {
       <Route index element={<LandingPage />} />
       <Route path="/Puzzle/Selection" element={<Verify><PuzzleSelectionPage /></Verify>} />
       <Route path="/Puzzle/1" element={<Verify><Puzzle1Page /></Verify>} />
+      <Route path="*"></Route>
```
Comment (Blackkirby72) [inline client/PageRoutes.jsx]: `<Route path="*" />`

you can just close it right away ^

~~thats my only thing ill let you guys review it~~

**Review by prewe5294 — APPROVED** [review]
looks good now

**Comment by Maxwell1455** [comment]
Smooth

**Review by Maxwell1455 — APPROVED** [review]
Smooth

**Pull request merged**